flyte.config

```diff
@@ -0,0 +1,11 @@
+[sdk]
+workflow_packages=workflows.getting_started
```

Comment:
lets call the folder something other than getting started
`myapp.workflows`

Reply:
done